src/test/java/it/pagopa/pn/cucumber/steps/SharedSteps.java

```diff
@@ -548,7 +548,7 @@ public String getFirstGroupByPa(String settedPa) {
         return id;
     }
 
-    public String getLastGroupByPa(String settedPa) {
+    public String getLastGroupIdByPa(String settedPa) {
```

Comment:
In questo caso non restituirei errore (Assertions.assertTrue(count >= 2)) nel caso in cui sia presente un solo gruppo attivo, che è un caso possibile, salvo che non ti serva per un motivo particolare.

Reply:
Viene lanciata eccezzione perchè vi desidera ottenere un gruppo diverso dal "first", se è presente un solo gruppo i test falliscono